src/thunderbots/software/ai/primitive/catch_primative.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <geom/angle.h>
+#include "primitive.h"
+
+class CatchPrimitive : public Primitive
+{
+public:
+    static const std::string PRIMITIVE_NAME;
+
+    /**
+     *
+     * Creates a new Catch Primative
```

Comment:
It's helpful to also have a brief description of what the Primitive does. Eg "Moves the robot into the ball's trajectory, using to dribbler to catch the ball and gain control, provided the robot is already in the general direction of the ball's travel"